.golangci.yml

```diff
@@ -9,13 +11,19 @@ run:
 linters:
   fast: false
   enable:
-  - gofmt
+  - errcheck
+  - exportloopref
+  - gocritic
+  - gofumpt
   - goimports
-  - revive
-  - govet
   - gomodguard
+  - govet
   - misspell
-  - exportloopref
+  - revive
+  - unconvert
+  - unparam
+  - unused
+  - whitespace
```

Comment:
can we also add `gosec`
https://github.com/securego/gosec

Reply:
It’s added !